s3/replication/common/tests/system/config/config.yaml

```diff
@@ -7,6 +7,8 @@ target_bucket_name: targetbucket # Buckets does exist on server
 object_name_prefix: foo_
 object_tag_name: user-tag
 object_tag_value: user-tag-value
+offset: -1
```

Comment:
please make it a place holder.
e.g. <range-read-offset>, <range-read-length>
same for all places.

Reply:
done